### DISCUSSION TITLE
How to optimize the lattice parameters for a PBC structure?

### DISCUSSION THREAD

**Discussion by TyBalduf** [grimme-lab/xtb#858] [Q&A]
What settings need to be used to optimize the lattice parameters along with the molecular coordinates? It seems like the default optimization engine currently can't do this, but it seems like engine=inertial should be able to. However, when I run with this, I seem to get a `step length` of 0 (and similarly a `cell power` and `cell speed` of 0) for every optimization step. 

Have I been trying the wrong sorts of systems (I tried some graphene sheets, as well as just a repeating methane atom) or do I just need to specify some keyword?

**Comment by awvwgk** [comment]
For using xTB methods with PBC, I recommend to look into DFTB+ which provides periodic optimizers and has support for xTB methods via the tblite library. The support for PBCs in xtb is pretty limited at the moment.
